fee/src/lib.rs

```diff
@@ -82,11 +67,11 @@ fn calculate_signature_fee(
     signature_count.saturating_mul(lamports_per_signature)
 }
 
-struct SignatureCounts {
-    pub num_transaction_signatures: u64,
-    pub num_ed25519_signatures: u64,
-    pub num_secp256k1_signatures: u64,
-    pub num_secp256r1_signatures: u64,
+pub struct SignatureCounts {
+    num_transaction_signatures: u64,
+    num_ed25519_signatures: u64,
+    num_secp256k1_signatures: u64,
+    num_secp256r1_signatures: u64,
```

Comment:
Trying to think if these should be public or not.

We will want to support the type `SanitizedTransactionView` (this is before we resolve addresses).

Using the `From` approach, we are forced to implement `From` either here or in `transaction-view` library. If private fields, forced to do it here (i.e. add dependency).
Seems we have more control as fee team if we add `transaction-view` dependency here?

Reply:
```
91 | impl<D: TransactionData> From<&SanitizedTransactionView<D>> for SignatureCounts {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `SignatureCounts`
   |
   = note: upstream crates may add a new impl of trait `solana_svm_transaction::svm_message::SVMMessage` for type `agave_transaction_view::transaction_view::TransactionView<true, _>` in future versions
```